README.md

```diff
@@ -83,6 +83,15 @@ within `kafka/ConsumerPositiveComprehensiveIT.java` to cover as many of
 the [transformation rules](https://github.com/companieshouse/filing-history-delta-consumer/blob/df6ee016f916e303474034ace5c3cc346b50d441/src/main/resources/transform_rules.yml)
 as possible.
 
+### Running Comprehensive CSV ITests via Intellij
```

Comment:
Thanks for this. I forgot to add this to the general integration test setup notes